services/leecher/leecher/listener.py

```diff
@@ -28,12 +28,16 @@
 import shotgun_api3
 
 # TODO: remove hash in future since it is only used as backward compatibility
+# filters on only real events, starting with `Leeched`, not on `Recreated`
```

Comment:
Either I don't understand the reasoning, or I don't understand the implementation. If we want to filter to specific topics, why we don't specify that in the `topics` filter?

Reply:
BTW I didn't find `filter` in graphql, that is some generic graphql filtering option?

Reply:
I removed this weirdness and replaced by specific subtopic.